scripts/tagging/parseversion.py

```diff
@@ -121,3 +121,10 @@ def getMajorMinorPatch( self ):
     """ :returns: tuple of Major, Minor, Patch """
     patch = 0 if self.patch is None else self.patch
     return self.major, self.minor, patch
+
+
+def getVersionComp(version):
+  parsed = Version(version)
+  major, minor, patch = parsed.getMajorMinorPatch()
+  pre = parsed.pre if parsed.pre else 0
+  return 100 * 100 * 100 * major + 100 * 100 * minor + 100 * patch + pre
```

Comment:
```suggestion
  return (major, minor, patch, pre)
```

and tuple comparison could do the rest. But this is really up to taste

Reply:
This tuple is much nicer than all the multiplications! Let me try that.